README.md

```diff
@@ -1,12 +1,29 @@
 # ui-components
 C2FO core ui components
 
+## Installing
+
+`npm install --save c2fo-ui-components`
+
 ## Documentation
 
 http://c2fo.github.io/ui-components/docs/
 
 ## Developer
 
+### Build Project
+
+`grunt build` - Build distribution version of the source code and runs the test suite against it
+
+### Source Bundle
+
+`grunt webpack:dev` - Watcher for webpack source files
+
+### Run Test Suite
+
+`grunt karma:dev` - Watcher for karma tests
+
+### Github Pages / Documentation
 This project uses git submodules and requires extra parameters for cloning to begin work.
 
 ```bash
```

Comment:
@vikeen and I chatted, I think the's still thinking over it. I don't think there is a need to do this branch inside a branch business. 

We can have the gh-pages branch get auto updated via a grunt task when we do a release.